trace/exporter_type.go

```diff
@@ -33,6 +33,16 @@ func (t ExporterType) MarshalJSON() ([]byte, error) {
 	return []byte(`"` + t.String() + `"`), nil
 }
 
+func (t *ExporterType) UnmarshalJSON(b []byte) error {
```

Comment:
Should we add some test vectors for marshal and unmarshal?

Reply:
Done